### PR TITLE
Completed LC129

### PR DESCRIPTION
**Pull request by alaneappen514**

### CODE REVIEW

File: alanEappen/assignments/treesDFS/lc129/lc129.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+]
+
+
+  dfs(root, '')
+  return sum 
+}
+edge cases:
+if there is no root return -1
+
+*/
```
Comment (knrodriguez) [inline alanEappen/assignments/treesDFS/lc129/lc129.js:49]: Great approach! :100:

File: alanEappen/assignments/treesDFS/lc129/lc129.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    }
+
+    if (node.right) {
+      dfs(node.right, num)
+    }
+  }
+  dfs(root, '')
+  return sum
+};
+
```
Comment (knrodriguez) [inline alanEappen/assignments/treesDFS/lc129/lc129.js:85]: Don't forget to write it twice to build your muscle memory!

File: alanEappen/assignments/treesDFS/lc129/lc129.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+
```
Comment (knrodriguez) [inline alanEappen/assignments/treesDFS/lc129/lc129.js:87]: Missing test case that matches edge cases.

**Review by knrodriguez — APPROVED** [review]
